app/models/agent.rb

```diff
@@ -154,4 +157,8 @@ def combined_search_terms
   def update_unknown_past_rdv_count!
     update_column(:unknown_past_rdv_count, rdvs.status(:unknown_past).count)
   end
+
+  def to_s
+    "#{last_name} #{first_name}"
```

Comment:
Maaaais pourquoi “nom prénom” au lieu de “prénom nom”?

```suggestion
    "#{first_name} #{last_name}"
```

Reply:
Ah oui, pour le tri dans les colonnes et liste de recherche, c'est mieux. Et je ne sais plus où est utilisé ce `to_s` alors je vais accepter ta proposition et on verra bien :)